tensorrt_llm/_torch/pyexecutor/_util.py

```
@@ -1256,15 +1265,23 @@ def _create_kv_cache_manager(
                         "using legacy MTP path")
             use_replay = False
 
-        # Replay Philox uses PTX cvt.rs.f16x2.f32 which needs sm >= 100.
+        # Replay Philox uses PTX cvt.rs.f16x2.f32 which needs 100 <= sm < 120.
         # Flashinfer has a SW fallback at any SM.
         if (stochastic_rounding
                 and mamba_params.mamba_ssm_cache_dtype == torch.float16
-                and sm < 100):
-            logger.info("Replay kernel Philox requires sm >= 100; "
+                and sm < 100 or sm in (120, 121)):
```

Ruff (0.15.12)

[warning] Parenthesize `a and b` expressions when chaining `and` and `or` together, to make the precedence clear

Parenthesize the `and` subexpression

(RUF021)